src/unix/linux_like/android/mod.rs

```diff
@@ -78,6 +78,14 @@ s! {
         pub l_pid: ::pid_t,
     }
 
+    pub struct flock64 {
+        pub l_type: ::c_short,
+        pub l_whence: ::c_short,
+        pub l_start: ::off64_t,
+        pub l_len: ::off64_t,
+        pub l_pid: ::pid_t,
+    }
```

Comment:
`l_start` and `l_end` have the wrong types on some android targets.

Reply:
Should be `__kernel_loff_t` and not `off64_t`.

Reply:
You're right, I looked at glibc here instead of google's code. https://android.googlesource.com/platform/bionic/+/refs/heads/master/libc/kernel/uapi/asm-generic/fcntl.h#161

Reply:
BTW, not really relevant but it seems like the sizes for `loff_t` etc. can be different per 32/64 bits depends on if the macro is enabled, so i'm not sure what to make of it 
https://android.googlesource.com/platform/bionic/+/refs/heads/master/libc/include/sys/types.h#100
https://android.googlesource.com/platform/bionic/+/master/docs/32-bit-abi.md

Reply:
You can just define `loff_t` differently for the different android targets.